api-extractor/testInputs/example4/src/NamespaceExport.ts

```diff
@@ -0,0 +1,65 @@
+import MyClass from './MyClass';
+
+/**
+ * This is a test for the namespace API type. It should be supported 
+ * and appear in the *api.ts file. At the moment we only support 
+ * a table of primitive types.
+ * 
+ * @public
+ */
+export namespace NamespaceExport {
```

Comment:
>namespace [](start = 7, length = 9)

You could also add a real-world test this by converting RushConstants to a namespace:

[https://github.com/Microsoft/web-build-tools/blob/master/rush/rush-lib/src/RushConstants.ts](https://github.com/Microsoft/web-build-tools/blob/master/rush/rush-lib/src/RushConstants.ts)